git-filter-repo

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
The commit message would need both an area prefix ("filter-repo: use python ...") and a Signed-off-by, but I can't take this change: see https://github.com/newren/git-filter-repo/issues/4#issuecomment-545030369.  We could add documentation that people need to change it, or even make it so if someone sets some variable then 'make install' will do the switch as part of the installation steps.

Reply:
Ugh, thanks for the link, finally an explanation! I looked all over the place trying to understand why a fresh python 3.8 install failed so miserably, but without luck... I guess "python vs python3 shebang on windows" aren't great Google keywords :)